src/pages/costDetails/costDetails.styles.ts

```diff
@@ -84,36 +93,80 @@ export const listViewOverride = css`
   ${global_Color_100.name}: ${global_Color_light_100.value};
   ${global_Color_200.name}: ${global_Color_light_200.value};
 
+  & * {
+    // outline: 1px solid blue;
+  }
+
+  & .list-group-item-heading,
+  & strong {
+    font-weight: ${global_FontWeight_normal.value};
+  }
+
   & .list-view-pf-main-info {
     display: block;
   }
+
+  & .list-view-pf-body {
+    display: flex;
+    justify-content: space-between;
+  }
+
   & .list-view-pf-description {
     display: inline-flex;
-    margin-left: 1rem;
+    margin-left: ${global_spacer_lg.value};
+  }
+
+  & .list-group-item-header {
+    display: flex;
+    flex: 1;
   }
-  & .list-view-pf-additional-info {
-    float: right;
+
+  & .list-view-pf-main-info {
+    flex: 1;
   }
-  .list-view-pf-expand {
+
+  & .list-view-pf-checkbox {
     float: left;
-    margin-right: 1rem;
   }
-  .list-view-pf-checkbox {
-    float: left;
+
+  & .list-view-pf-expand {
+    min-width: 1.5rem;
   }
+
   .list-view-pf-additional-info strong {
-    position: relative;
+    display: block;
+    text-align: right;
   }
+
   .list-view-pf-additional-info span {
     font-size: ${global_FontSize_xs.value};
     color: ${global_disabled_color_100.value};
-    top: 1rem;
-    position: relative;
-    right: 3.5rem;
   }
-  .list-group .list-group-item:first-child {
+
+  .list-group-item {
+    padding: ${global_spacer_lg.value} ${global_spacer_xl.value};
+    border-bottom: 2px solid #f2f2f2;
+  }
+
+  .list-group-item:first-child {
     border-top: 0;
-    padding-left: 2.25rem;
+    padding: 0.55rem ${global_spacer_lg.value} ${global_spacer_xs.value} 3.5rem;
+    font-size: ${global_FontSize_sm.value};
+  }
+
+  & .list-group-item-container {
+    padding-top: 1rem;
+  }
+
+  .list-view-pf-expand-active {
+    box-shadow: ${global_BoxShadow.value};
+    z-index: 1;
+    background-image: linear-gradient(
+      to right,
+      ${global_primary_color_100.value},
+      ${global_primary_color_100.value} 3px,
+      transparent 3px
+    );
```

Comment:
With PR #126, I've introduced patternfly-additions.css. We need that in order to show filter results in the toolbar. Thus, many of the style overrides have been removed.

I'm thinking it would be best to have the filter results in place, first? Then, we can apply styling to make that look more like PF4.